### PR TITLE
fixed pop method elements deleted

### PR DESCRIPTION
**Pull request by ElenaBorisenkoO**
please check the pop method

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -26,17 +26,14 @@ MyArray.prototype.push = function(...rest) {
 };
 
 MyArray.prototype.pop = function() {
-  let x = this.elements[this.elements.length - 1];
+  const x = this[this.length - 1];
+  const newelements = new MyArray();
```
Comment (AndrZakhara) [inline src/index.js]: newelements must be newElements

**Review by AndrZakhara — CHANGES_REQUESTED** [review]
fix var name

**Review by Jarosic — APPROVED** [review]
Ок)

**Review by AndrZakhara — APPROVED** [review]
great job

**Pull request merged**